src/openapi_server/apis/reading_api.py

```diff
@@ -21,9 +23,26 @@
 
 from openapi_server.connect import client_for_basic_auth
 
+from requests_toolbelt.multipart import decoder
+
 router = APIRouter()
 
 
+def decode_multipart_response(response):
+    multipart_data = decoder.MultipartDecoder.from_response(response)
+    return multipart_data.parts[0].text
+
+
+def unpack_list(xpath_list):
```

Comment:
Can we get a test for this function as well?

Reply:
Test added.